apple-pie/styles/style.css

```diff
@@ -0,0 +1,72 @@
+h1 {
+  color: white;
+  background-image: url(/apple-pie/images/apple-pie.jpg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100%;
+  height: 250px;
+  text-align: center;
+  padding-top: 200px;
+  font-size: 70px;
+}
+
```

Comment:
A good practice would be to add at the beginning of the stylesheet a set of reset rules, in order to remove all default browser styles (this is why you have a top margin on your header for example).
You can use the one from Eric Meyer: https://meyerweb.com/eric/tools/css/reset/